main.c

```diff
@@ -0,0 +1,17 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+int main(void)
+{
+  int number;

```

Comment:
Try to always initialize local variables, if you did not initialize them you can get the issue when a variable has a garbage value

Reply:
Done